src/Sql/Platform/Mysql/SelectDecorator.php

```diff
@@ -50,7 +50,8 @@ protected function processLimit(
             return;
         }
         if ($parameterContainer) {
-            $parameterContainer->offsetSet('limit', $this->limit, ParameterContainer::TYPE_INTEGER);
+            $param_prefix = $this->processInfo['paramPrefix'];
```

Comment:
Please use camelCase variable names instead of underscore_separated style.

Reply:
Yes, i think case with 1 query is valid, i will add it)